include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -3523,8 +3523,6 @@ namespace internal
             else
               {
                 // case 5: default vector access
-                AssertDimension(n_face_orientations, 1);
-
```

Comment:
This check is removed, since it is now checked in `FEFaceEvaluation::evaluate()`.